packages/discovery/src/discovery/handlers/user/ArrayFromOneEventHandler.ts

```diff
@@ -25,6 +26,7 @@ export const ArrayFromOneEventHandlerDefinition = z.strictObject({
   ),
   invert: z.optional(z.boolean()),
   ignoreRelative: z.optional(z.boolean()),
+  topics: z.optional(z.array(z.union([z.string(), z.null()]))),
```

Comment:
why `z.null`?

Reply:
the idea is that if you want to specify only `topic2`, you need to provide `null` as `topic1`